frontend/app/src/App.tsx

```diff
@@ -1613,6 +1627,8 @@ export class App extends PureComponent<Props, State> {
             availableThemes: this.props.theme.availableThemes,
             addThemes: this.props.theme.addThemes,
             hideFullScreenButtons: false,
+            hostConfig,
+            setHostConfig: this.setHostConfig,
```

Comment:
Is this necessary to have/pass down in `LibContext`? Took a quick glance through the full prototype and doesn't look like any of the child components would need to/should change the host config settings?

Reply:
Oh, good catch. I just blindly followed the other context variables, but in this case there is indeed no need for a setter function 👍

Reply:
I removed it